apps/src/templates/rubrics/EvidenceLevels.jsx

```diff
@@ -12,6 +12,7 @@ export default function EvidenceLevels({
   radioButtonCallback,
   submittedEvaluation,
   isStudent,
+  isAutosaving,
```

Comment:
It looks like you passed in a value for this from `LearningGoals` but not `LearningGoal`. Intentional or bug?

Reply:
Yes, that's a bug. LearningGoal will eventually be replaced by LearningGoals, but until then I'm trying to push bug fixes to both.